index.js

```diff
@@ -105,13 +106,16 @@ const run = async () => {
         // Create Issue
         createdIssues.push({ title, body, labels })
       } else {
-        title = `${issueTitlePrefix}${new Date().toTimeString()}`
-        createdIssues[0].title = title
-
+        counter++
+        createdIssues[0].title = `${issueTitlePrefix}${title}`;
```

Comment:
`title` already contains the prefix. Also the linter doen't like the semicolon and thinks `title` shoud be `const`.

The change looks good otherwise.

Reply:
Good call, thank you for double-checking.

Reply:
Oh, and I just noticed that the first pushed issue _already_ has the intended title ;-)

Reply:
Could you have one more quick look before I merge?